app/models/appointment.rb

```diff
@@ -0,0 +1,19 @@
+class Appointment < ApplicationRecord
+  belongs_to :user
```

Comment:
Yes that thought crossed my mind a lot. Is it worth it having to refer to the class name directly? I think so.